kubernetes/loculus/values.yaml

```diff
@@ -1,5 +1,6 @@
 environment: server
 robotsNoindexHeader: false
+ncbiGatewayUrl: "https://ncbi-temp-proxy.api.taxonium.org/datasets/v2"
```

Comment:
Shouldn't this rather go to the preview config and not be set here as a default?

Reply:
(Just for later, when this works and should be merged: this config is probably best to go under `ingest.configFile`?)

Reply:
I don't think we have an ingest configuration that sits outside of organism configuration? IMO this should

Reply:
We could introduce one, maybe something as simple as this for now?

```
ingest:
  ncbiGatewayUrl: ...
```

Otherwise, it seems very unclear to me what `ncbiGatewayUrl` applies to/means.

Reply:
(implemented, thanks!)